### DISCUSSION TITLE
Thank you for your contributions in November 2024

### DISCUSSION THREAD

**Discussion by MisRob** [learningequality/kolibri#12924] [Community]
**We're grateful for all the wonderful contributions we received in the past month. During November:**

- @BabyElias guided contributors in continuing her GSOC design system table component, and started working on introducing it to another place in Kolibri
- @EshaanAgg improved developer experience in Studio and started working on new features for the design system table component
- @GarvitSinghal47 finished implementing a new GitHub action that automatically updates a spreadsheet containing a list of community pull requests. This helps us respond to volunteers in a timely manner and more.
- @emmanuel-ferdman fixed broken references in the Kolibri Developer Documentation
- @lokesh-sagi125 removed deprecated drop shadows from the design system, and updated Kolibri to use the focus trap design system component.
- @m3tal10 made sure that thumbnail images in Kolibri will be loaded lazily
- @RONAK-AI647 fixed issues with the visibility of the focus ring in Firefox on the design system website
- @rparadowski made an action that sends us Slack notifications re-usable and shared it across five repositories, allowing us to maintain it much more easily
- @Sahil-Sinha-11 fixed issues with the keyboard navigation in the dropdown menu design system component
- @thesujai added ancestor information to Kolibri’s summary and session log exports
- @yeshwanth235 improved documentation of the select design system component

💛 It's joyful to see many of you continuing to volunteer.  Let me welcome all new contributors and thank to everyone who has some work in progress. I wish you all happy transition to the new year and hope to see many of you in 2025.

P.S. Keep an eye on the forums in the second half of January, where I'd like to look back at 2024 and mention what's ahead in 2025.

---

**_If you haven’t already, we’d love for you to [check out the Kolibri Contributors Community](https://github.com/learningequality/kolibri/discussions/12156)—a growing group of volunteers who regularly contribute to Learning Equality's repositories. It’s a great way to connect, with an option to have your contributions featured on social media platforms like LinkedIn._**

**Comment by RosannaDomino** [comment]
Thank you all! The Learning Equality staff is very aware of and grateful for your contributions. Thank you for supporting education through your coding.
